multicluster/config/overlays/member/manager_command_patch.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: controller
+  namespace: system
+spec:
+  template:
+    spec:
+      containers:
+        - command:
+            - /antrea-mc-controller
+            - member
+          name: manager
```

Comment:
ditto

Reply:
Could you please rename the `manger` to `antrea-mc-controller`

Reply:
Sure